### PR TITLE
Add batch backwards mode to helper script

### PR DESCRIPTION
**Pull request by makramkd**
Add batch backwards mode to store many blockhashes in a single transaction using `BatchBlockhashStore.storeVerifyHeader`.

### CODE REVIEW

File: core/scripts/vrfv2/testnet/main.go [hunk 1]
```diff
@@ -123,6 +126,72 @@ func main() {
...
+
+			fmt.Println("using nonce:", nonce)
+
+			fmt.Println("estimating gas")
+
+			gasLimit, err := ec.EstimateGas(context.Background(), ethereum.CallMsg{
+				From:     owner.From,
+				To:       &batchbhsaddr,
+				Gas:      10_000_000, // put a big number here, we will probably estimate lower for batch sizes < 75
+				GasPrice: new(big.Int).Set(gp).Mul(gp, big.NewInt(2)),
```
Comment (Nic0S) [inline core/scripts/vrfv2/testnet/main.go:171]: nit: can be simplified to just new(big.Int).Mul(gp, big.NewInt(2))  I think

File: core/scripts/vrfv2/testnet/main.go [hunk 2]
```diff
@@ -123,6 +126,72 @@ func main() {
...
+			helpers.PanicErr(err)
+
+			nonce, err := ec.PendingNonceAt(context.Background(), owner.From)
+			helpers.PanicErr(err)
+
+			fmt.Println("using nonce:", nonce)
+
+			fmt.Println("estimating gas")
+
+			gasLimit, err := ec.EstimateGas(context.Background(), ethereum.CallMsg{
```
Comment (Nic0S) [inline core/scripts/vrfv2/testnet/main.go:167]: Why is gas estimation + using the generic SendTransaction necessary vs just calling the function on the typed batched_bhs golang type?

Comment (makramkd) [reply core/scripts/vrfv2/testnet/main.go:167]: Ah good point. I think initially I was going to try bumping the nonce manually and sending e.g 5 transactions at a time but I think that's a bit more error prone without the BPTXM. Will switch to using the go wrapper.

File: core/scripts/vrfv2/testnet/main.go (hunk 1 above)
Comment (makramkd) [reply core/scripts/vrfv2/testnet/main.go:171]: You're absolutely correct!

File: core/scripts/vrfv2/testnet/main.go (hunk 2 above)
Comment (Nic0S) [reply core/scripts/vrfv2/testnet/main.go]: You should be able to set the nonce manually with the go wrapper too by setting the owner.Nonce field

**Review by Nic0S — APPROVED** [review]

**Pull request merged**